### ISSUE TITLE
[Tarea 4] [P3] MAE

### ISSUE THREAD

**Issue by emyskog** [UC-IIC2613/Syllabus#204]
Al fijarse si el MAE es menor que 800, debe ser el MAE de el validation set o del training set?

**Comment by diegomendieta** [comment]
Tiene que ser el set de test, ya que es el que mide el rendimiento real de tu modelo evitando el sesgo en el que se incurre al afinar hiperparámetros para mejorar el rendimiento en el set de validación.

**Comment by emyskog** [comment]
pero tipo 810 esta bien? o tiene que ser menor sí o sí que 800?

**Comment by diegomendieta** [comment]
Tiene que ser menor que 800.

**Comment by matiasmsm** [comment]
O sea, ¿ no es mlp.fit(X_train, y_train, epochs=15, batch_size=32, verbose=1) el fit?

Yo pensaba que era el MAE de training y que despues se probaba con el set de test para el mejor modelo.

Siento que el enunciado genera muchas confusiones :(

**Comment by diegomendieta** [comment]
@MatiasMingo sí, ese es el fit. Tienes que observar cómo avanza el entrenamiento en el set de training y de validación, ajustar los hiperparámetros de manera tal que se minimicen MSE y MAE en validación, y finalmente evaluar tu mejor modelo en test. Si el MAE es menor que 800 en test, entonces se considerará correcta tu respuesta.
